docs/cues/cue-playback.md

```diff
@@ -176,42 +176,64 @@ Settings](playback-options.md#fader-tab) in playback [Options](playback-options.
+
+You can also configure the key profile so that the handle buttons 
 can be used to release the playback using [Key Profiles](../system-settings/key-profiles.md).
 
-Channels will always release with a fade time, this defaults to 2
-seconds but you can change it using \[Master Release Time\] in the
-Release menu. You can also set an individual release time for the
-playback from the [Release tab](playback-options.md#release-tab) of playback [Options](playback-options.md).
+Channels will always release with a fade time, this defaults to 2 seconds but you can change it using \[Master Release Time\] in the Release menu. You can also set an individual release time for the playback from the [Release tab](playback-options.md#release-tab) of playback options.
 
--   You can instantly kill a playback by pressing \<Avo\> + the **Select**
-    button of the playback.
+-   You can instantly kill a playback (ignoring any fade out times) by pressing \<Avo\> + the **Select**
+    button of the playback. This can also be useful for Cue Lists which (depending on their options) may stay fired even when the fader is lowered to zero.
 
--   You can set a mask so that only certain attributes release from the
-    playback when the playback is killed. This uses the 
-    [\[Release Mask\] option](playback-options.md#release-mask) in 
-    playback [Options](playback-options.md).
-    Normally all attributes will remain in the state last used in the
-    playback - *for cues which trigger strobing or other effects this
-    might not be what you want*.
+-   You can release a running playback by pressing \<Release\>, then the **Select** button of the playback to be released. This uses a [Temporary Release Mask](#temporary-release-mask) as described below.   
+
+-   You can **Release All** running playbacks by double pressing \<Release\>. The user setting [Release Priority](../system-settings/user-settings.md#release) sets which playback priority levels will be released. You can protect against unintended Release All by setting the Release Priority to Low - this would only release playbacks with Low priority, if you had any set up.
+
+
+Release Mask
+------------
+
+You can configure which attributes will release from a playback when it is killed using the **Release Mask**. There is a **Global**
+release mask which is the default setting, or you can set a **Local** release mask which overrides the global mask for a particular playback.
+If you start a release using the \<Release\> button then a **Temporary** release mask is used which just affects the current release operation.
+
+### Global Release Mask
+
+If no local release mask is set for the playback, attributes release
+according to this mask when the playback is killed. The default is for no attributes to release, so all LTP attributes will remain at their last value.
+
+Set the Global Release Mask from the **Release Menu** by pressing \<Release\> then \[Global Release Mask\]. The
```

Comment:
I believe this is also accessible via User Settings

Reply:
Don't think it is, or can't see it anyway. It is not in the Release options - just Master Time

Reply:
Interesting. You are correct. This is surprising as underneath it is a user setting I believe... 🐛

Reply:
filed #87680